src/app/core/services/gene.service.spec.ts

```diff
@@ -43,16 +43,17 @@ describe('Service: Gene: TestBed', () => {
         expect(gcglSpy).toHaveBeenCalledTimes(2);
     });
 
-    it('should update the previous gene with the current gene', () => {
-        const upgSpy = spyOn(geneService, 'updatePreviousGene').and.callThrough();
-        geneService.setCurrentGene(mockGene1);
-        expect(geneService.getCurrentGene()).toEqual(mockGene1);
-        expect(geneService.getPreviousGene()).toEqual(undefined);
-
-        geneService.updatePreviousGene();
-        expect(upgSpy).toHaveBeenCalled();
-        expect(geneService.getPreviousGene()).toEqual(mockGene1);
-    });
+    // AG-293 - This fails intermittently
+    // it('should update the previous gene with the current gene', () => {
+    //     const upgSpy = spyOn(geneService, 'updatePreviousGene').and.callThrough();
+    //     geneService.setCurrentGene(mockGene1);
+    //     expect(geneService.getCurrentGene()).toEqual(mockGene1);
+    //     expect(geneService.getPreviousGene()).toEqual(undefined);
+    //
+    //     geneService.updatePreviousGene();
+    //     expect(upgSpy).toHaveBeenCalled();
+    //     expect(geneService.getPreviousGene()).toEqual(mockGene1);
+    // });
```

Comment:
Why not just delete this?

Reply:
I can move the test case into the related JIRA ticket if having it in here makes you sad!

Reply:
I used the built-in 'xit' feature to suppress the test from execution, so this is better now.